cavityFaultDisplay.py

```diff
@@ -27,66 +27,87 @@ def __init__(self, parent=None, args=None, macros=None):
         headerLayout = QHBoxLayout()
         statusheaderLabel = QLabel()
         statusheaderLabel.setText("Status")
+        statusheaderLabel.setMaximumWidth(100)
+        statusheaderLabel.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding)
+        statusheaderLabel.setAlignment(Qt.AlignCenter)
         statusheaderLabel.setStyleSheet("text-decoration: underline")
 
         nameheaderLabel = QLabel()
         nameheaderLabel.setText("Name")
-        nameheaderLabel.setMinimumSize(200, 30)
-        nameheaderLabel.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Maximum)
+        nameheaderLabel.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding)
+        nameheaderLabel.setAlignment(Qt.AlignLeft)
         nameheaderLabel.setStyleSheet("text-decoration: underline")
 
         codeheaderLabel = QLabel()
         codeheaderLabel.setText("Code")
-        codeheaderLabel.setMinimumSize(30, 30)
-        codeheaderLabel.setSizePolicy(QSizePolicy.Maximum, QSizePolicy.Maximum)
+        codeheaderLabel.setMaximumWidth(50)
+        codeheaderLabel.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.MinimumExpanding)
+        codeheaderLabel.setAlignment(Qt.AlignHCenter)
         codeheaderLabel.setStyleSheet("text-decoration: underline")
 
-        headerLayout.setAlignment(Qt.AlignVCenter | Qt.AlignLeft)
-
         headerLayout.addWidget(codeheaderLabel)
         headerLayout.addWidget(nameheaderLabel)
         headerLayout.addWidget(statusheaderLabel)
-        headerLayout.setSpacing(50)
 
+        headerLayout.setAlignment(Qt.AlignVCenter | Qt.AlignLeft)
         verticalLayout.addLayout(headerLayout)
 
         for fault in faults.values():
             horizontalLayout = QHBoxLayout()
 
             statusLabel = QLabel()
-            statusLabel.setStyleSheet("font-weight: bold")
+            statusLabel.setMaximumWidth(100)
+            statusLabel.setMaximumHeight(28)
+            statusLabel.setMinimumHeight(28)
             statusLabel.setSizePolicy(QSizePolicy.MinimumExpanding,
                                       QSizePolicy.MinimumExpanding)
+            statusLabel.setAlignment(Qt.AlignCenter)
+
+            # statusLabel.setFrameStyle(QFrame.Box | QFrame.Plain)
+
+            # statusLabel.setLineWidth(2)
```

Comment:
If we don't need these lines, maybe delete them?

Reply:
fixed!